docker/env/.streamer.env

```diff
@@ -4,7 +4,7 @@ SUBSTRATE_URI=wss://rpc.polkadot.io
 
 KAFKA_URI=broker:9092
 
-DB_HOST=db
+DB_HOST=postgres
```

Comment:
Changed to postgres, as we have set hostname as postgres in the docker-compose.yml file